packages/vx-heatmap/package.json

```diff
@@ -33,9 +34,10 @@
     "access": "public"
   },
   "peerDependencies": {
-    "react": "^15.0.0-0 || ^16.0.0-0"
+    "react": "^16.3.0-0"
   },
   "dependencies": {
+    "@types/classnames": "^2.2.9",
```

Comment:
do we need to add `"@types/react": "*"` here?

Reply:
yes, good catch 🤦‍♂done.